packages/web-component/src/components/read-along-component/read-along.tsx

```diff
@@ -548,7 +548,7 @@ export class ReadAlongComponent {
     if (this.svgOverlay) {
       // either with svg overlay
       this.animateProgressWithOverlay();
-    } else {
+    } else if (play_id) {
```

Comment:
What's this about? The Cypress tests seem to fail now about something related to scrolling.

Reply:
It's an attempt to fix another bug that's been intermittently popping up in the Cypress tests.

This has been nagging me for a while: every few CI runs, I have to click rerun failed jobs to see them pass. Del and I worked on that yesterday and we thought this would fix it, but apparently not, apparently it just moved the symptom around a bit. We'll have to keep digging I guess.

Reply:
Confirmed, I just clicked "rerun failed jobs" is CI passed this time. I'll take this commit out of this PR, though, since it's not the quick fix I hoped it would be. I'm really looking forward to that bug being addressed!